android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -75,7 +75,7 @@
     public static final String CLOSE_EVENT = "close-twilio-activity";
     public static final String SEND_EVENT = "send-twilio-event";
 
-    private final int CONTROLS_ANIMATION_DELAY = 3000;
+    private final int CONTROLS_ANIMATION_DELAY = 6000;
```

Comment:
6 seconds?

Reply:
increase because user has to interact twice (open the menu and select the option)